baldrick/plugins/tests/test_github_towncrier_changelog.py

```diff
@@ -55,3 +75,16 @@ def test_changelog_present(self, app):
 
         for message in messages.values():
             assert message['conclusion'] == 'success'
+
+    @pytest.mark.xfail(
+        sys.platform.startswith('win'),
+        reason='process_towncrier_changelog returns failure on Windows')
```

Comment:
Is it the slash?

Reply:
I copied this decorator from the one above.

Reply:
I wouldn't think it is the slash, because this path comes down from github it's not a local path? but it probably is somehow.